shared/js/content-scripts/content-scope-messaging.js

```diff
@@ -10,6 +10,22 @@ function getSecret () {
 async function init () {
     const secret = await getSecret()
 
+    window.addEventListener('sendMessage', event => {
```

Comment:
I was wondering why we need to add the listener again here, since it's included in content-scope-scripts. I looked into it, I think that's since we don't have a listener in [content-scope-scripts/inject/chrome-mv3.js](https://github.com/duckduckgo/content-scope-scripts/blob/main/inject/chrome-mv3.js) like we do in [content-scope-scripts/inject/chrome.js](https://github.com/duckduckgo/content-scope-scripts/blob/main/inject/chrome.js). I think it would be better to add this listener to content-scope-scripts instead.